src/components/settings/data/hooks.js

```diff
@@ -56,6 +63,108 @@ export const useLinkManagement = (enterpriseUUID) => {
+          dark: primaryColor.darken(COLOR_LIGHTEN_DARKEN_MODIFIER),
+          textColor: getA11yTextColor(primaryColor),
+        },
+        secondary: {
+          regular: secondaryColor,
+          light: secondaryColor.lighten(COLOR_LIGHTEN_DARKEN_MODIFIER),
+          dark: secondaryColor.darken(COLOR_LIGHTEN_DARKEN_MODIFIER),
+          textColor: getA11yTextColor(secondaryColor),
+        },
+        tertiary: {
+          regular: tertiaryColor,
+          light: tertiaryColor.lighten(COLOR_LIGHTEN_DARKEN_MODIFIER),
+          dark: tertiaryColor.darken(COLOR_LIGHTEN_DARKEN_MODIFIER),
+          textColor: getA11yTextColor(tertiaryColor),
+        },
+      };
+    },
+    [branding],
+  );
+
+  if (!isDefined(brandColors)) {
+    return null;
+  }
+
+  const styles = ({
+    key: 'styles',
+    styles: (`
+      .hero-brand {
+        background-color: ${brandColors.secondary.regular.hex()} !important;
+        border-color: ${brandColors.tertiary.regular.hex()} !important;
+        color: ${brandColors.secondary.textColor.hex()} !important;
+      }
+      .hero-brand h1 {
+        color: ${brandColors.secondary.textColor.hex()} !important;
+      }
+      .btn-brand {
+        background-color: ${brandColors.secondary.regular.hex()} !important;
+        border-color: ${brandColors.secondary.regular.hex()} !important;
+        color: ${brandColors.secondary.textColor.hex()} !important;
+      }
+      .btn-brand:hover {
+        background-color: ${brandColors.secondary.dark.hex()} !important;
+        border-color: ${brandColors.secondary.dark.hex()} !important;
+      }
+      .btn-brand:focus:before {
+        border-color: ${brandColors.secondary.regular.hex()} !important;
+      }
+      .btn-primary {
+        background-color: ${brandColors.primary.regular.hex()} !important;
+        border-color: ${brandColors.primary.regular.hex()} !important;
+        color: ${brandColors.primary.textColor.hex()} !important;
+      }
+      .btn-primary:hover {
+        background-color: ${brandColors.primary.dark.hex()} !important;
+        border-color: ${brandColors.primary.dark.hex()} !important;
+      }
+      .btn-brand-primary {
+        background-color: ${brandColors.primary.regular.hex()} !important;
+        border-color: ${brandColors.primary.regular.hex()} !important;
+        color: ${brandColors.primary.textColor.hex()} !important;
+      }
+      .btn-brand-primary:hover {
+        background-color: ${brandColors.primary.dark.hex()} !important;
+        border-color: ${brandColors.primary.dark.hex()} !important;
+      }
+      .btn-brand-primary:focus:before {
+        border-color: ${brandColors.primary.regular.hex()} !important;
+      }
+      .bg-brand-primary {
+        background-color: ${brandColors.primary.regular.hex()} !important;
+      }
+      .border-brand-primary {
+        border-color: ${brandColors.primary.regular.hex()} !important;
+      }
+    `),
+  });
+
+  return styles;
```

Comment:
Because this is a re-usable hook, it may make sense to ensure `styles` is memoized (`useMemo`) such that it doesn't cause unnecessary re-renders (i.e., right now, every time the hook runs it will have a new object reference and if it were used in a `useEffect` by a consumer, for example, it could cause that consumer to run the `useEffect` more times than intended.